met-web/src/components/engagement/listing/index.tsx

```diff
@@ -252,7 +252,11 @@ const EngagementListing = () => {
                         >
                             <SearchIcon />
                         </PrimaryButton>
-                        <SecondaryButton onClick={() => setAdvancedSearchOpen(!advancedSearchOpen)}>
+                        <SecondaryButton
+                            data-testid="engagement/listing/advancedSearch"
+                            name="advancedSearch"
+                            onClick={() => setAdvancedSearchOpen(!advancedSearchOpen)}
+                        >
                             {advancedSearchOpen ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />} Advanced Search
```

Comment:
would be nice to do what MUI did with the arrow
![image](https://user-images.githubusercontent.com/91914654/217033241-eee12906-0753-4e66-8411-8af11ca42884.png)

Reply:
updated, thanks